### PR TITLE
add NETStandard1.0 project

### PR DESCRIPTION
**Pull request by acple**
add new project targeting NETStandard1.0.
but hasn't added to nuspec yet.

### CODE REVIEW

File: src/ContextFreeTasks.NETStandard/ContextFreeTasks.NETStandard.csproj [hunk 1]
```diff
@@ -6,7 +6,11 @@
     <Compile Include="**\*.cs" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="NETStandard.Library" Version="1.6.1" />
+    <PackageReference Include="System.Resources.ResourceManager" Version="4.3.0">
```
Comment (ufcpp) [inline src/ContextFreeTasks.NETStandard/ContextFreeTasks.NETStandard.csproj]: This commit is not required. I prefer the `NETStandard.Library` package.

Comment (acple) [reply src/ContextFreeTasks.NETStandard/ContextFreeTasks.NETStandard.csproj]: yes, netcoreapp application and netstandard library project are based on NETStandard.Library package.
but some other platforms (targeting net, mono, or etc, used like PCL) aren't dependent to NETStandard.Library.
if use package depend to NETStandard.Library, ALL OF NetStandard packages are installed, and almost all are not used.

Comment (ufcpp) [reply src/ContextFreeTasks.NETStandard/ContextFreeTasks.NETStandard.csproj]: For mono, PCL, etc., we must create another project which refers the shared project anyway. This csproj is dedicated to netstandard, thus there is no reason why this csproj avoids `NETStandard.Library`.

Comment (acple) [reply src/ContextFreeTasks.NETStandard/ContextFreeTasks.NETStandard.csproj]: ah I understand it.
yes, when this project is targeting netcoreapp only, this commit can be discarded.

**Pull request merged**